Snippets/wnl.sqlpromptsnippet

```diff
@@ -0,0 +1 @@
+WITH (NOLOCK) 
```

Comment:
I'll check when I'm in the office--was sure I just uploaded the snippet file....